service/worker/deletenamespace/reclaimresources/workflow.go

```diff
@@ -166,27 +168,27 @@ func deleteWorkflowExecutions(ctx workflow.Context, params ReclaimResourcesParam
 
 		if isAdvancedVisibility {
 			ctx3 := workflow.WithActivityOptions(ctx, ensureNoExecutionsAdvVisibilityActivityOptions)
```

Comment:
Any reason to have both client side retry and activity server side retry?

Reply:
I don't want to introduce one more child workflow here, it is already pretty complex. Aslo in this case there is no backoff and I wouldn't even call it "rerty". It is mostly "try to do it with 3 attempts".